src/callbacks.rs

```diff
@@ -146,7 +146,7 @@ pub trait FindexCallbacks<Error: std::error::Error + CallbackError, const UID_LE
         new_chain_table_items: EncryptedTable<UID_LENGTH>,
     ) -> Result<(), Error>;
 
-    /// Returns all locations among the ones given that do not exist anymore.
```

Comment:
Question coding style: les commentaires avec trois '/' sont voulus ?

Reply:
Yes, `///` is the Rust way to comment function / trait / struct (docblocks) `//` are for inline comment inside code.